frontend/src/components/editCard/EditComment.tsx

```diff
@@ -1,23 +1,46 @@
 import React from "react";
+import Marked, { MarkedOptions } from "marked";
+import DOMPurify from "dompurify";
 import { Card } from "react-bootstrap";
 import { Link } from "react-router-dom";
 
-import { Edit_findEdit_comments as Comment } from "src/graphql/definitions/Edit";
+import { Edit_findEdit_comments_user as User } from "src/graphql/definitions/Edit";
 import { formatDateTime, userHref } from "src/utils";
 
 const CLASSNAME = "EditComment";
 
 interface Props {
-  comment: Comment;
+  comment: string;
+  date: string;
+  user?: Pick<User, "name">;
 }
 
-const EditComment: React.FC<Props> = ({ comment }) => (
+DOMPurify.addHook("afterSanitizeAttributes", (node: Element) => {
+  if (node.tagName === "A") {
+    node.setAttribute("target", "_blank");
+    node.setAttribute("rel", "noopener nofollow");
+  }
+});
+
+const options: MarkedOptions = {
+  gfm: true,
+  breaks: true,
+};
+
+const EditComment: React.FC<Props> = ({ comment, date, user }) => (
   <Card className={CLASSNAME}>
-    <Card.Body className="pb-0">{comment.comment}</Card.Body>
+    <Card.Body className="pb-0">
+      {/* eslint-disable-next-line react/no-danger */}
+      <div
```

Comment:
The warning still came up when I used `yarn start`, this fixed it.

```suggestion
      <div
        /* eslint-disable-next-line react/no-danger */
```